src/filtering/filters.spec.ts

```diff
@@ -195,6 +260,56 @@ describe("filters (incoming)", () => {
       )
     ).toEqual([Filter.INCOMING]);
   });
+  it("is still INCOMING when there are pending review comments, and reviewer in reviewer team", () => {
+    const env = buildTestingEnvironment();
+    expect(
+      getFilteredBucket(
+        env,
+        "kevin",
+        NOTHING_MUTED,
+        fakePullRequest()
+          .author("fwouts")
+          .seenAs("kevin")
+          .teams({
+            team: ["kevin"],
+          })
+          .reviewRequested([], ["team"])
+          .addReview("kevin", "PENDING")
+          .build()
+      )
+    ).toEqual([Filter.INCOMING]);
+  });
+  it("is MUTED when the PR is muted until next update and the author did not add new comments or reviews", () => {
+    const env = buildTestingEnvironment();
+    expect(
+      getFilteredBucket(
+        env,
+        "kevin",
+        {
+          mutedPullRequests: [
+            {
+              repo: {
+                owner: "zenclabs",
+                name: "prmonitor",
+              },
+              number: 1,
+              until: {
+                kind: "next-update",
+                mutedAtTimestamp: 100,
+              },
+            },
+          ],
+        },
+        fakePullRequest()
+          .author("fwouts")
+          .seenAs("kevin")
+          .reviewRequested(["kevin"])
+          // Another user posted a review after we muted.
+          .addReview("dries", "CHANGES_REQUESTED", 200)
+          .build()
+      )
+    ).toEqual([Filter.MUTED]);
+  });
   it("is MUTED when the PR is muted until next update and the author did not add new comments or reviews", () => {
```

Comment:
This test currently has the same description as the one above, but it's a different scenario (when a team is requested). Please rename.

Reply:
done